components/bl-multiple-select-component/src/multiple-select-field/input.js

```diff
@@ -0,0 +1,31 @@
+import { Chip } from './chip';
+
+export function Input(props) {
+  const { multipleSelectId, multipleSelectValue, typeOfMultipleSelect } = props;
```

Comment:
What about remove 'multiple' prefix? We know from context about select is multiple, and we have no other selects.